actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -55,12 +63,26 @@ def call(env)
 
       response
     rescue Exception => exception
+      invoke_interceptors(request, exception)
       raise exception unless request.show_exceptions?
       render_exception(request, exception)
     end
 
     private
 
+    def invoke_interceptors(request, exception)
+      backtrace_cleaner = request.get_header('action_dispatch.backtrace_cleaner')
+      wrapper = ExceptionWrapper.new(backtrace_cleaner, exception)
+
+      @interceptors.each do |interceptor|
```

Comment:
:shrug: I just think it would make sense to use the direct variable we will iterating on, instead of creating a shadow variable, that we won't even be changing.